tmail-backend/jmap/extensions/src/main/java/com/linagora/tmail/james/jmap/EmailAddressContactFactory.java

```diff
@@ -140,19 +140,20 @@ private XContentBuilder generateSetting(int nbShards, int nbReplica, int waitFor
                             .startObject("analysis")
                                 .startObject("analyzer")
                                     .startObject("contact_analyzer")
-                                    .field("tokenizer", "contact_tokenizer")
+                                        .field("tokenizer", "contact_tokenizer")
                                     .endObject()
                                 .endObject()
                             .endObject()
                             .startObject("tokenizer")
                                 .startObject("contact_tokenizer")
-                                .field("type", "ngram")
-                                .field("min_gram", 2)
-                                .field("max_gram", 20)
-                                .startArray("token_chars")
-                                    .value("letter")
-                                    .value("digit")
-                                .endArray()
+                                    .field("type", "ngram")
+                                    .field("min_gram", 2)
+                                    .field("max_gram", 20)
```

Comment:
Should be a trigram 3/3 -> @chibenwa comment on this has not been resolved

Reply:
I've just commited a fixed up to this items and some others above.